roles/foreman_installer/tasks/trust_github.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: 'Add SSH_AUTH_SOCK to env_keep in sudoers'
+  lineinfile:
+    dest: /etc/sudoers
+    state: present
+    regexp: SSH_AUTH_SOCK
+    line: 'Defaults env_keep += "SSH_AUTH_SOCK"'
+
+- name: 'Ensure known_hosts file exists'
+  copy:
```

Comment:
Would the file module not be more appropriate?

Reply:
@ehelms https://github.com/theforeman/forklift/pull/525#discussion_r138335184